src/plugins/es_ui_shared/static/forms/components/index.ts

```diff
@@ -19,6 +19,7 @@ EuiCodeEditor (for instance, explicitly).
 Importing here is a way of preventing a more sophisticated solution to this problem since we want to, eventually,
 migrate all code editors over to Monaco. Once that is done, we should remove this import.
  */
+import 'react-ace';
```

Comment:
`ace`/`react-ace` is required to be imported before `brace` modes. This was (unintentionally?) relying on the `react-ace` import in `@elastic/eui` that no longer  exists.

Perhaps there is a better way to handle this?